src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,26 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
 
+    public static void main(String[] args) throws Exception {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book bobsBook = new Book();
+        bobsBook.setTitle("Head First Java");
+        bobsBook.setPrice(new BigDecimal(350));
+        bookDao.create(bobsBook);
+        bookDao.findById(1L);
```

Comment:
Id must be generated by the database. Do not hardcode it in the code
```suggestion
        bookDao.findById(bobsBook.getId());
```

Reply:
Ok. Thank you